### PR TITLE
feat: Show Network Fees in USD on L2

### LINKED ISSUE #907: Synth Exchange: Show Network Fees in USD on L2
### Subject of the issue
Synth Exchange when connected to L2 still shows the Gas Price, which is relatively obsolete (always is 0.001 on L2). Instead, we should show the Network Gas Fee value USD (similar to #850). Currently it is only visible in the tooltip. Switching this makes the tooltip obsolete.

### Expected behaviour
On L2, show USD value of Gas Costs, in the screenshot example below e.g. `$0.37`.

### Actual behaviour
Screen Shot 2022-05-27 at 15 08 45

### Deliverables
- When connected to L2, make sure to show Network Gas Fee in USD. 
- Remove the tooltip
- When connected to L1, nothing needs to be changed. UI should still provide the "Max Fee" value in Gwei. For clarification please add a "Gwei" label behind the Gas value on L1.

### Rewards
25 SNX

### PR DESCRIPTION
**Pull request by lohba**
## Description

- Network gas fee is now displayed in $ on L2
- USD Value field has been removed for L1 and L2 and tool tip has been removed from L2
- When connected to L1, "Gwei" label has been added 

## Related issue

PR closes #907
## Screenshots (if appropriate):
L2
Screen Shot 2022-06-06 at 1 16 09 PM

L1
Screen Shot 2022-06-06 at 1 19 04 PM

### CODE REVIEW

**Review by platschi — APPROVED** [review]
Works great now and as expected. Much cleaner UI now! Thank you!

Tested on both L1 and L2 Mainnet.

File: sections/shared/components/GasPriceSelect/GasPriceSelect.tsx [hunk 1]
```diff
@@ -29,16 +23,20 @@ const GasPriceSelect: FC<GasPriceSelectProps> = ({ gasPrices, transactionFee, ..
...
 	const gasPrice = gasPrices ? parseGasPriceObject(gasPrices[gasSpeed]) : null;
 
-	const gasPriceItem = hasCustomGasPrice ? (
-		<span data-testid="gas-price">{formatNumber(customGasPrice, { minDecimals: 4 })}</span>
-	) : (
-		<span data-testid="gas-price">{formatNumber(gasPrice ?? 0, { minDecimals: 4 })}</span>
+	const gasPriceItem = (
+		<span data-testid="gas-price">
+			{isL2
+				? formatCurrency(Synths.sUSD, transactionFee ?? 0, { sign: '$', maxDecimals: 1 })
```
Comment (Tburm) [inline sections/shared/components/GasPriceSelect/GasPriceSelect.tsx:35]: One nitpick, I would prefer this doesn't show $0.00 when the input isn't valid. The other inputs show `-` when the inputs are not complete.

**Review by Tburm — APPROVED** [review]
@platschi Approving code, but your call on whether we should make the quick change to show `-` instead of `$0.00` with invalid inputs

**Comment by Tburm** [comment]
@lohba I am not seeing the max fee show on L1 when I make my first selection. 

To test:
* Set wallet to Ethereum mainnet
* Refresh exchange page
* Select quote currency
* Input base currency amount [no max fee here]

Then...
* Change quote currency
* Input base currency amount [max fee shows up now]

I think you need to wrap the `gasPriceItem` in a `useMemo` or something similar to make sure it updates when the inputs change.

**Review by platschi — APPROVED** [review]

**Review by Tburm — APPROVED** [review]

**Pull request merged**